tests/testthat/test-add_difference.tbl_summary.R

```diff
@@ -614,12 +620,14 @@ test_that("ordering in add_difference.tbl_summary() with paired tests", {
         by = am,
         include = mpg
       ) |>
-      add_difference(test = ~"paired.t.test", group = id) |>
+      add_difference(test = ~"paired.t.test", group = id)
+
+    tbl |>
       modify_column_hide(all_stat_cols()) |>
       as.data.frame()
-  )
-  expect_snapshot(
-    mtcars |>
+  })
+  expect_snapshot({
```

Comment:
Great way of doing the snapshot testing, i.e. to add the whole generating code block